src/coins.cpp

```diff
@@ -998,6 +1014,8 @@ double CCoinsViewCache::GetPriority(const CTransaction &tx, int nHeight) const
         }
     }
 
+    // TZE: priority code is being removed concurrently, so not updating here.
```

Comment:
Until it is, I think this will cause problems in mining transactions with only TZE inputs.